guides/README.md

```diff
@@ -1,6 +1,6 @@
-# Ansible Lightbulb - Ansible Engine
+# Ansible Lightbulb Guides
```

Comment:
This was done to introduce and enforce a clear and consistent naming.
We had workshops in lightbulb, and later added guides. The gh pages need to be clear what is what.